uiowa/uiowa
# Page listblock

Resolves #6578 

# How to test

```
ddev blt ds --site sandbox.uiowa.edu && ddev drush @sandbox.local uli
```
- Add a new page or use an existing page. Add a page list blocks (Add block > More > Pages) to the layout.
- Run through the test scenarios below with one page list block.
- Add a second page list block and run through the scenarios again with two list blocks on the page, ensuring that neither block is affected by other one.
- Add one of the existing list blocks (article or people) to the page and test again, ensuring that the existing list block is not affected by the page list blocks.
- Independently test the article or people list block on a separate page to ensure that there are no regressions.

## Test scenarios
- Setting one filter value.
- Exposing one filter with no value set.
- Exposing one filter with a value set.
- Exposing one filter with no value set and setting the value for the other filter but not exposing it.
- Exposing both filters with no values set.
- Exposing both filters and one filter has a value set.
- Exposing both filter with values set.
- Combine the above settings with a pager.
- Combine the above settings with different sort settings.
- Test different combinations of LB styles.
